.gitignore

```diff
@@ -17,4 +17,8 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 dist/
-.cache/
\ No newline at end of file
+.cache/
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/*.cfg.yml
+!.elasticbeanstalk/*.global.yml
```

Comment:
Are you using elastic beanstalk or docker? You should consider removing files that are redundant.